### PR TITLE
Fix runner dependency

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-installation.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Apr  1 11:39:32 UTC 2015 - jreidinger@suse.com
+
+- fix dependencies to not force different proposal store to not
+  import it
```
Comment (ancorgs) [inline package/yast2-installation.changes]: I actually don't understand what the fix is about. Can you please try to rephrase it?

Comment (jreidinger) [inline package/yast2-installation.changes]: maybe simple fix dependencies in proposal_runner?

Comment (ancorgs) [inline package/yast2-installation.changes]: More understandable, yes.

**Comment by ancorgs** [comment]
LGTM

**Pull request merged**